frontend/src/components/CourseDescriptionPanel/CourseAttributes/CourseAttributes.tsx

```diff
@@ -83,6 +83,18 @@ const CourseAttributes = ({ course, courseCapacity }: CourseAttributesProps) =>
             </S.Link>
           ) : null
         },
+        {
+          title: 'UNSW Course Outline',
+          content: studyLevel ? (
+            <S.Link
+              href={`https://www.unsw.edu.au/course-outlines/course-outline#year=2024&term=Term%201&deliveryMode=Multimodal&deliveryFormat=Standard&teachingPeriod=${terms[0]}&deliveryLocation=Kensington&courseCode=${code}&activityGroupId=1`}
```

Comment:
> &term=Term%201
Seems like the term is still hardcoded here. Not sure if teaching period can be hardcoded as it is currently or if it needs something special?

Reply:
Thank you lucas, 
I'm working on it that t1 it would be 201 t2 it would be 202 and t3 it would be 203, then summer course it would be term=summer 

also not sure if the teaching period should be like that, because if we open up a course like comp4128 which is only offered during t3, and click the link it would show not present because the course outline would be released once the course starts

Reply:
I wonder if we need to start scraping course outlines properly, and then only show actual outline URLs we have...